cmake/OpenCVFindLibsPerf.cmake

```diff
@@ -6,14 +6,17 @@
 if(WITH_LAPACK)
   find_package(LAPACK)
   if(LAPACK_FOUND)
-      find_path(LAPACK_INCLUDE_DIR "lapacke.h")
-      if(LAPACK_INCLUDE_DIR)
+    find_path(LAPACKE_INCLUDE_DIR "lapacke.h")
+    if(LAPACKE_INCLUDE_DIR)
+      find_path(CBLAS_INCLUDE_DIR "cblas.h")
+      if(CBLAS_INCLUDE_DIR)
         set(HAVE_LAPACK 1)
-        ocv_include_directories(${LAPACK_INCLUDE_DIR})
+        ocv_include_directories(${LAPACKE_INCLUDE_DIR} ${CBLAS_INCLUDE_DIR})
         list(APPEND OPENCV_LINKER_LIBS ${LAPACK_LIBRARIES})
       endif()
-  endif(LAPACK_FOUND)
-endif(WITH_LAPACK)
+    endif()
+  endif()
+endif()
 
```

Comment:
should not we reflect lapack code dependency via CPP code? E.g.:

```
#if defined HAVE_LAPACK && defined HAVE_CBLAS
lapack_gemm_c, etc.
#endif
``` 

Because this CMake code disallows lapack usage without cblas.